chain/pool/txpool.go

```diff
@@ -87,6 +89,34 @@ func (tp *TxnPool) AppendTxnPool(txn *transaction.Transaction) error {
 	return nil
 }
 
+func (tp *TxnPool) totalClaim(sender []byte) (common.Fixed64, error) {
+	amountToClaim := common.Fixed64(0)
+	var err error
+	tp.NanoPayTxs.Range(func(_, v interface{}) bool {
```

Comment:
Currently txnpool.NanoPayTxs stores map txnHash -> txn, so for every nanopay txn added, all existing nano txns in txn pool needs to be traversed and the cost is O(N^2) where N is the # of nano pay txn put into txn pool. What about changing it to map sender -> (map txnHash -> txn)? txnpool.TxLists is already doing it now.

Reply:
I understand what you mean, though I'm not sure if we can use txnpool.TxLists as it is (it relies on nonce-ordering and also have concept of orphans, both of which nanopay doesn't have), might need to come up with something else.

Reply:
What I mean is that we don't use the current txn pool (because nanopay txn does not have a nonce), but just change what is stored in NanoPayTxs.

Reply:
there is no total claim counting in pool anymore